src/py/_pyodide/_base.py

```diff
@@ -211,6 +211,24 @@ class CodeRunner:
 
         The flags to compile with. See the documentation for the built-in
         :external:py:func:`compile` function.
+
+    Examples
+    --------
+    >>> from pyodide.code import CodeRunner
+    >>> source = "1 + 1"
+    >>> code_runner = CodeRunner(source)
+    >>> code_runner.compile()
+    <_pyodide._base.CodeRunner object at 0x113de58>
+    >>> code_runner.run()
+    2
+    >>> my_globals = { "y": 20 }
+    >>> my_locals = { "y": 5 }
```

Comment:
Can you run black on it somehow? Probably no spaces after { no?

Also it would be better to define a differently named variable in globals and locals says x and y. Then un the code just do x + y and it should work as far as I understand.